test/Pixel.rb

```diff
@@ -13,13 +13,21 @@ def test_hash
     hash = nil
     assert_nothing_raised { hash = @pixel.hash }
     assert_not_nil(hash)
-    assert_equal(1_385_501_952, hash)
+    if IM_7
+      assert_equal(1_385_502_079, hash)
```

Comment:
Why did the pixel hash change between versions? This seems like something that ought to be stable.

Reply:
This is changed because of the value of the alpha channel:

```c
    hash  = ScaleQuantumToChar(pixel->red)   << 24;
    hash += ScaleQuantumToChar(pixel->green) << 16;
    hash += ScaleQuantumToChar(pixel->blue)  << 8;
#if defined(IMAGEMAGICK_7)
    hash += ScaleQuantumToChar(pixel->alpha);
#else
    hash += ScaleQuantumToChar(pixel->opacity);
#endif
```

Reply:
If we change `pixel` to use `alpha` instead of opacity we should have the same value for both IM7 and IM6.